application/pom.xml

```diff
@@ -170,6 +170,11 @@
             <version>1.64</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
i don't think that it will be needed after putting the files in the correct directory